seqr/views/apis/family_api.py

```diff
@@ -49,15 +49,6 @@ def family_page_data(request, family_guid):
     response = {
         'samplesByGuid': {s['sampleGuid']: s for s in samples}
     }
-    rna_type_samples = {
```

Comment:
this was causing a performance problem in this endpoint, which is why the performance bug is also solved by theis PR